src/main/java/seedu/address/model/event/Event.java

```diff
@@ -0,0 +1,92 @@
+package seedu.address.model.event;
+
+import static seedu.address.commons.util.CollectionUtil.requireAllNonNull;
+
+import java.util.Objects;
+
+import seedu.address.commons.util.ToStringBuilder;
+
+/**
+ * Represents a Person in the address book.
```

Comment:
Just some minor typos, person should be Event.
```suggestion
 * Represents a Event in the address book.
```